sdk/storage/azure-storage-file-share/azure/storage/fileshare/_directory_client.py

```diff
@@ -463,6 +463,9 @@ def rename_directory(
         :keyword str file_permission_key:
             Key of the permission to be set for the directory.
             Note: Only one of the file-permission or file-permission-key should be specified.
+        :keyword file_permission_format:
+            Specifies the format in which the permission is returned. Acceptable values are "SDDL" or "binary".
```

Comment:
Let's add "If not specified, SDDL will be the default". Also if you change the type to Literal like I suggested, you won't need the "Acceptable values are..." part since it will be part of the type.